packages/insomnia-app/app/ui/components/graph-ql-explorer/graph-ql-explorer-type.js

```diff
@@ -77,12 +77,13 @@ class GraphQLExplorerType extends React.PureComponent<Props> {
 
     // $FlowFixMe
     const fields = type.getFields();
+    const fieldKeys = Object.keys(fields).sort();
```

Comment:
```suggestion
    const fieldKeys = Object.keys(fields).sort((a, b) => a.localeCompare(b));
```

I think it would be more appropriate to use `localeCompare` here, since we're comparing names. This will ensure that non-ascii characters and such are handled correctly.